.github/workflows/copilot-setup-steps.yml

```diff
@@ -0,0 +1,23 @@
+name: "Copilot Setup Steps"
+
+# Allow testing of the setup steps from your repository's "Actions" tab.
+on: workflow_dispatch
+
+jobs:
+  # The job MUST be called `copilot-setup-steps` or it will not be picked up by Copilot.
+  copilot-setup-steps:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    # You can define any steps you want, and they will run before the agent starts.
+    # If you do not check out your code, Copilot will do this for you.
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Restore solution
+        env:
+          # prevent GitInfo errors
+          CI: false
```

Comment:
I see you copied this from https://github.com/dotnet/aspire/pull/9414, however I'm not sure that applies here.  I'm guessing that was needed for Aspire's build where it was enabling some more targets with `-ci` and it expected something to be injected into it's build to provide those.